hyped.code-workspace

```diff
@@ -35,5 +35,8 @@
     "editor.detectIndentation": false,
     "C_Cpp.default.cppStandard": "c++11",
     "C_Cpp.default.cStandard": "c11",
+    "files.associations": {
```

Comment:
Did you change or it was done by your pc?

Reply:
Must have been my PC, don't remember changing it